onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -124,10 +129,142 @@ def _parse_line(line):
 
     return (scalefacs, zpoints)
 
+# Adding ability to record intermediate graph outputs
+selected_types = ['Conv', 'MatMul'] # node types to extend
+
+def augment_graph(model):
+    '''
+    Adds ReduceMin and ReduceMax nodes to all Conv and MatMul nodes in
+    model and ensures their outputs are stored as part of the graph output
+        parameter model: loaded FP32 ONNX model to quantize
+        return: augmented ONNX model
+    '''
+    added_nodes = []
+    added_outputs = []
+    for node in model.graph.node:
+        if node.op_type in selected_types:
+            input_name = node.name
+            # Adding ReduceMin nodes
+            reduce_min_name = input_name + '_ReduceMin'
+            reduce_min_node = onnx.helper.make_node('ReduceMin', [input_name + ':0'],
+                            [reduce_min_name + ':0'], reduce_min_name, keepdims=0)
```

Comment:
if you choose the right name for this output a lot of your processing in the data collection stage will be simpleif you use ode.output[0] + "rmin<or something similar>" then during data collection stage you already know which outputs rmin is this

Reply:
Fixed.